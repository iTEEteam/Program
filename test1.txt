loadmap testmap1.txt
buy tower 2-1
buy tower 2-5
buy gem 2-1 speed
buy gem 2-1 range
buy gem 2-1 damage
buy gem 2-5 enemy elf
make hobbit 1-1
update 2
make elf 1-1
update 2
sell 2-1
exit