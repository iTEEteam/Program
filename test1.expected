map testmap1.txt loaded
Tower0 created
Tower1 created
SpeedGem0 created
RangeGem0 created
DamageGem0 created
EnemyTypeGem0 created
Hobbit0 created
update
Hobbit0 moves
Tower0 shoots Hobbit0
Hobbit0 damage 40
update
Hobbit0 moves
Tower0 shoots Hobbit0
Hobbit0 damage 40
Tower1 shoots Hobbit0
Hobbit0 damage 20
Hobbit0 died
Elf0 created
update
Elf0 moves
Tower0 shoots Elf0
Elf0 damage 40
update
Elf0 move
Tower0 shoots Elf0
Elf0 damage 40
Tower1 shoots Elf0
Elf0 damage 60
Elf0 died
Tower0 sold 80 mana
VEGE
